src/pages/SigningOfficialResearchers.js

```diff
@@ -20,13 +19,7 @@ export default function SigningOfficialResearchers() {
       try {
         setIsLoading(true);
         const soUser = await User.getMe();
-        const soPromises = await Promise.all([
-          User.list(USER_ROLES.signingOfficial),
-          User.getUnassignedUsers()
-        ]);
-        const researcherList = soPromises[0];
-        const unregisteredResearchers = soPromises[1];
```

Comment:
Just a note that the naming here was incorrect, these were not unregistered users, they were registered users with no institution selection.